_posts/2022-03-15-blog-qterminal-bookmarks.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: QTerminal Bookmarks reloaded
+slug: qterminal-bookmarks-reloaded
+date: mar 15, 2022, 14:54:49
+promoted: true
+categories: blog
+---
+
+QTerminal had always had a quite unique "bookmarks" feature but the setup was not quite intuitive: the file had
+to be created first, it had a hardcoded section containing not very useful bookmarks, filtering never worked
+ and so on. Probably few users used it, as some issue never were reported.
+ Now - among many other bugfixes in QTerminal and his dropdown twin -
+ we dedicated some time to make this feature more attractive and usable to users.
+
+ It can be used in a similar way as <code>~/.bash_aliases</code> - a real easy way to use
+  longer commands or hard ones to remember. Commands can be ordered in groups.
```

Comment:
"longer commands or hard ones to remember" → "longer commands or those that are hard to remember"